spec/features/access_indexing_spec.rb

```diff
@@ -100,7 +100,7 @@ def choose_rights(view:, download: nil, location: nil, cdl: false)
   end
 
   # It takes a few milliseconds for the rights update to take
-  expect(page).to have_content(view_label(view: view, location: location, cdl: cdl))
+  expect(page).to have_text(view_label(view:, location:, cdl:))
```

Comment:
Instead of inconsistent usage of has/ve_text and has/ve_content, pick one and be consistent.